docs/zoneddatetime.md

```diff
@@ -175,8 +175,8 @@ Instead of using `Temporal.ZonedDateTime` with an offset time zone, it may be ea
 
 The `overflow` option works as follows, if `item` is an object:
 
-- In `'constrain'` mode (the default), any out-of-range values are clamped to the nearest in-range value.
-- In `'reject'` mode, the presence of out-of-range values will cause the function to throw a `RangeError`.
+- In `'constrain'` mode (the default), any out-of-range values are clamped to the nearest in-range value (after assuming extension of eras over arbitrary years to substitute `era` and `eraYear` with appropriate values for the `item`).
+- In `'reject'` mode, the presence of out-of-range values (after assuming extension of eras over arbitrary years to substitute `era` and `eraYear` with appropriate values for the `item`) will cause the function to throw a `RangeError`.
```

Comment:
There's an inconsistency which should probably be addressed: [Temporal.ZonedDateTime.from](https://tc39.es/proposal-temporal/docs/zoneddatetime.html#from) documentation includes quotes in code-formatted expected string values (`'constrain'`, `'reject'`, etc.) which are absent from other documentation such as [Temporal.PlainDateTime.from](https://tc39.es/proposal-temporal/docs/plaindatetime.html#from) (`constrain`, `reject`, etc.). I don't know what conventions are preferred by the Temporal champions, and at any rate am leaving that aspect untouched by this PR.

Reply:
I don't really have a preference but would default to `'string'` if nobody else had a preference either. I've opened #2988 for this.